.github/workflows/prepare-release.yml

```
@@ -15,10 +12,12 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v3
 
-    - name: Extract current version
+    - name: Extract current version and scarb version
```

```suggestion
    - name: Extract current versions
```
since we're not saying current version of what, it thought this was simpler